web/src/components/pages/dashboard/challenge/weekly-challenge.tsx

```diff
@@ -131,7 +132,7 @@ export default function WeeklyChallenge({ isNarrow }: { isNarrow?: boolean }) {
                       key={index}
                       isNarrow={isNarrow}
                       avatarUrl={account.avatar_url}
-                      challengeTeam={'sap'} //account.challenge_team}
+                      logoUrl={challengeLogoUrls['sap']}
```

Comment:
Are all these logos right now pointing to SAP because we're waiting on the other two companies? Or is this just a placeholder?

Reply:
Yep!

> Note: the Mozilla and IBM svgs that I added are placeholders until we get updated assets from design.